requirements/edx/base.txt

```diff
@@ -319,7 +320,7 @@ django-mptt==0.13.4
     #   django-wiki
 django-multi-email-field==0.6.2
     # via edx-enterprise
-django-mysql==4.6.0
+django-mysql==4.7.0
```

Comment:
Added support for `Python 3.11` and `Django 4.1`. Dropped support for `Maria DB 10.2`. No other breaking change mentioned in the [CHANGELOG](https://github.com/adamchainz/django-mysql/blob/main/HISTORY.rst#470-2022-06-05)